brion/plugin/compartmentReportMap.cpp

```diff
@@ -394,6 +394,45 @@ floatsPtr CompartmentReportMap::loadFrame( const float time ) const
     }
 
     floatsPtr buffer( new floats( getFrameSize( )));
+    if( _load( buffer, keys, offsetMap ))
+        return buffer;
+    return floatsPtr();
+}
+
+floatsPtr CompartmentReportMap::loadNeuron( const uint32_t gid ) const
+{
+    if( !_readable )
+        return floatsPtr();
+
+    const size_t index = std::distance( _gids.begin(), _gids.find( gid ));
+    if( index >= _gids.size( ))
+        return floatsPtr();
+
+    const std::string& scope = _uri + std::to_string( gid ) + "_";
```

Comment:
style: drop the & for local variable which is not actually a ref? const std::string or const auto